doc/changelog/10-standard-library/14087-q-lemmas/qlemmas.rst

```diff
@@ -0,0 +1,2 @@
+Added lemmas: Qreduce_l, Qreduce_r, Qinject_Z_l, Qinject_Z_r, Qeq_from_parts, Qden_cancel, Qnum_cancel
```

Comment:
Please run `dev/tools/make-changelog.sh` to create this entry in the standard format.

Also, perhaps you could summarize the changes rather than just listing the new items?  And mention `QArith.v` so readers can easily find your additions.